src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/PublishContent.targets

```diff
@@ -25,7 +25,9 @@
     <CreateAzureContainer
       AccountKey="$(CloudDropAccessToken)"
       AccountName="$(CloudDropAccountName)"
-      ContainerName="$(ContainerName)" />
+      ContainerName="$(ContainerName)"
+      WriteOnlyTokenDaysValid="1"
+      ReadOnlyTokenDaysValid="1" />
```

Comment:
No, it's not causing a problem in the build.  It's just an annoying warning.  I'll pull this out of my change if you prefer, or leave it and file an issue.  I should know better than to try to cleanup unrelated annoyances in my PR's :)